app/adapters/application.js

```diff
@@ -35,6 +35,12 @@ export default HalAdapter.extend({
     } else {
       return error;
     }
+  },
+
+  deleteRecord: function(store, type, record){
```

Comment:
I don't actually use this deletion code anymore. Instead this flow goes through deprovision